frontend/src/app/profile-edit/index.tsx

```diff
@@ -0,0 +1,342 @@
+    console.log(
+      "Effect: formState.values: " + JSON.stringify(formState.values),
+    );
+  });
+
+  return (
+    <>
+      <h1>
+        <FormattedMessage id="profile.header"/>
+      </h1>
+
+      {loading && "Loading..."}
+      {!loading && (
+        <Form onSubmit={onFormSubmit} method="post">
+          {errorMessage && <Alert type="error">{errorMessage}</Alert>}
+          <Row paddingBottom={ROW_PADDING} paddingTop={BUTTON_PADDING}>
+            <Column columnWidth={COLUMN_WIDTH}>
+              <LayoutSet>
+                <Card>
+                  <h3>
+                    <FormattedMessage id="profile.edit.personalHeader"/>
+                  </h3>
+
+                  <InputWrapper text={errorsFields.firstName} isRequired={true}>
+                    <InputField
+                      inputProps={{
+                        id: "firstName",
+                        ...text("firstName"),
+                      }}
+                      a11yId="firstName"
+                      // fyi: <Label> doesn't have isRequired property, doesn't show
+                      // "*" to indicate that the field is required
+                      label={
+                        <FormattedMessage id="profile.firstName">
+                          {msg => <b>{msg}</b>}
+                        </FormattedMessage>
+                      }
+                      onChange={hangleUserChange}
+                      isRequired={true}
+                    />
+                  </InputWrapper>
+
+                  <InputWrapper text={errorsFields.lastName} isRequired={true}>
+                    <InputField
+                      inputProps={{
+                        id: "lastName",
+                        ...text("lastName"),
+                      }}
+                      a11yId="lastName"
+                      label={
+                        <FormattedMessage id="profile.lastName">
+                          {msg => <b>{msg}</b>}
+                        </FormattedMessage>
+                      }
+                      onChange={hangleUserChange}
+                    />
+                  </InputWrapper>
+
+                  <InputWrapper text={errorsFields.gender} isRequired={true}>
+                    <RadioGroupField
+                      {...radio("gender",
+                        {
+                          name:"gender",
+                          onChange: event => {
+                            const gender = event.target.value;
+                            formState.setField("gender", gender);
+                            return gender;
+                          },
+                        })}
+                      isHorizontal={true}
+                      a11yId="gender"
+                      label={
+                        <FormattedMessage id="profile.gender">
+                          {msg => <b>{msg}</b>}
+                        </FormattedMessage>
+                      }
+                      options={[
```

Comment:
These options are hardcoded, of course, they are pretty simple and should not change... but at least we should translate, can I take values from `locale.index.tsx` dictionary? [I can't use FormattedMessage component... ]
Is ugly?? Is better put somewhere a resolver in the BE?